spec/models/reference_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'rails_helper'
 
-RSpec.describe Reference, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+describe Reference do
+  it 'is valid with name address position' do
+    expect(build(:reference)).to be_valid
+  end
```

Comment:
Oh, if it doesn't have any validations then we don't need to test at all. Looks like it has a `belongs_to` association so maybe test for that instead.